src/cms-driver.ts

```diff
@@ -159,6 +159,21 @@ export class CMakeServerClientDriver extends CMakeDriver {
       log.error('Found no matching code model for the current build type. This shouldn\'t be possible');
       return [];
     }
+    const metaTargets = [{
```

Comment:
It would be possible to check the version of the cmake executable (`this.cmake`).

Reply:
What do you think about checking, whether the `hasInstallRule` field exists?

Reply:
I think this should do it, shouldn't it?